versions/1.21.5/src/main/java/at/hannibal2/skyhanni/features/foraging/MoongladeBeacon.kt

```diff
@@ -111,6 +115,15 @@ object MoongladeBeacon {
             upgradingStrength = true
         }
         currentServerTicks = 0
+        checkPants()
```

Comment:
this game really does lead to some of the funniest function names i've seen in a codebase